src/HS/UserBundle/Tests/Controller/DefaultControllerTest.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace HS\UserBundle\Tests\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+class DefaultControllerTest extends WebTestCase
+{
+    public function testIndex()
+    {
+        $client = static::createClient();
+
+        $crawler = $client->request('GET', '/');
+
+        $this->assertContains('Hello World', $client->getResponse()->getContent());
+    }
```

Comment:
@Reddaa peut être qu'il faut faire plus de test pour les cas non passant 🙏

Reply:
@ahaizoune  the FOSUserBundle is taking care of that for the module, besides in the default bundle I'll add what's needed 👍